meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -63,6 +63,9 @@ CONFIG_FASTBOOT_UUU_SUPPORT=y
 CONFIG_FASTBOOT_BUF_ADDR=0x80800000
 CONFIG_FASTBOOT_BUF_SIZE=0x03800000
 CONFIG_FSL_FASTBOOT_BOOTLOADER2=y
+CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY=y
+CONFIG_FSL_FASTBOOT_BOOTLOADER2_OFFSET=0x8a
```

Comment:
This should be the default value for imx6

Reply:
fixed